alipay/container-auto-tune
# add CI for project builder

# Which issue does this PR close?

Closes https://github.com/alipay/container-auto-tune/issues/4

# Rationale for this change
 Currently, this project has not ci to compile project when PR is created.



# What changes are included in this PR?
add ci.yaml


# Are there any user-facing changes?
No


# How does this change test
Given the ci.yaml is added.
When the collaborators submit PR to main branch.
Then the ci will started
And You can click  the "Actions"  to observe the "Java With Maven"
